decko-rails/lib/decko/rails.rb

```diff
@@ -0,0 +1,15 @@
+DECKO_RAILS_GEM_ROOT = File.expand_path('../../..', __FILE__)
+
+module Decko
+  module Rails
+    class << self
+      def root
+        ::Rails.root
+      end
+
+      def gem_root
```

Comment:
I finally get that now, sorry.  Then this goes away if we combine wagn and decko-rails into decko.  That now seems correct to me.